src/components/common/deleteResource.jsx

```diff
@@ -114,7 +116,9 @@ export const DeleteResourceButton = ({ objectId, disabled, overlayText, actionNa
     } else {
         return (
             <Button id={`delete-${objectId}`}
-                variant='danger'
+                variant={variant || 'danger'}
+                isDanger={!!variant}
```

Comment:
Shouldn't that be `{variant === 'danger'}`? (Also, why is that not the default in PF? Specifying both `variant="danger"` and `isDanger` seems redundant)

Reply:
I'm trying to work here basically with 2 types of button. Danger button, and Danger link button.

1. `<Button variant="link" isDanger>`:
![Screenshot from 2022-04-07 23-56-31](https://user-images.githubusercontent.com/42733240/162326798-7f3acf63-027b-43a6-bd15-a42175c75434.png)

2. `<Button variant="danger">`: 
![Screenshot from 2022-04-07 23-56-57](https://user-images.githubusercontent.com/42733240/162326812-fee9357a-b111-4e54-8d03-fb9f477f4973.png)

Reply:
But no I see that I programmed it a bit confusingly. Maybe instead of `variant` parameter being passed to `DeleteResourceButton`, I can have parameter `isLink`, and then this button could be:
`<Button variant={isLink ? 'link' : 'danger' isDanger={isLink} />`
That would probably be clearer.